### PR TITLE
Optimize performances on folder creation

### PR DESCRIPTION
**Pull request by ngilain**
[OO-MISC] Optimize performances on folder creation
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/298
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1907

### CODE REVIEW

**Comment by alavieille** [comment]
:+1:

**Comment by ngilain** [comment]
@alavieille : new optimisation added

File: MediaAdminModelBundle/Tests/EventListener/AddMediaFolderGroupRoleForFolderListenerTest.php [hunk 1]
```diff
@@ -47,41 +50,39 @@ public function testMethodPrePersistCallable()
      *
      * @dataProvider provideGroupAndSite
      */
-    public function testPostPersist(array $groups,  $site, $countMFGR)
+    public function testPostPersist(array $groups, $expectedPersistTime)
```
Comment (alavieille) [inline MediaAdminModelBundle/Tests/EventListener/AddMediaFolderGroupRoleForFolderListenerTest.php:52]: php doc

**Comment by alavieille** [comment]
else php doc :+1:

**Pull request merged**